src/main/java/com/aws/greengrass/security/CryptoKeySpi.java

```diff
@@ -20,4 +20,8 @@ KeyManager[] getKeyManagers(URI privateKeyUri, URI certificateUri)
     KeyPair getKeyPair(URI privateKeyUri, URI certificateUri) throws ServiceUnavailableException, KeyLoadingException;
 
     String supportedKeyType();
+
+    default String getKeyAlias(URI privateKeyUri, URI certificateUri) {
```

Comment:
How about implementing what I talked with you about last night? getCertificate.

Since I think we agreed that we don't care about the alias. All we want is the certificate.

Reply:
Changed.